frontend/src/components/Secrets/GSecretDialog.vue

```diff
@@ -332,7 +324,13 @@ export default {
       this.hide()
     },
     async submit () {
-      this.v$.$touch()
+      if (this.secretValidations.$invalid) {
+        await this.secretValidations.$validate()
+        const message = messageFromErrors(this.secretValidations.$errors)
+        this.errorMessage = 'There are input errors that you need to resolve'
+        this.detailedErrorMessage = message
+        return
+      }
       if (this.valid) {
```

Comment:
valid does not exist anymore. This if has to be removed. Currently it is not possible to create any secrets